### ISSUE TITLE
The illogical disappearance of sticky bombs.

### ISSUE THREAD

**Issue by twilightwanderer** [tgstation/tgstation#69537]
When designing faxes, I planned to be able to send sticky bombs along with the paper. I found this idea quite amusing. But in my planned fax update, I wanted to rework this mechanic to make it less "evil". While studying the sticky bomb mechanics, I came across an interesting flaw in their current implementation.

## Reproduction:

It is enough to trigger a qdel object to remove the bomb as well. From a purely technical point of view it is clear why this is necessary, but such a possibility opens up too easy a way to combat such bombs.
![dreamseeker_Zn9QBxq3sc](https://user-images.githubusercontent.com/88540658/187175802-4d84d0e0-2f33-4d0c-8cee-25c20c5119b2.gif)

**Comment by timothymtorres** [comment]
In line with #69484, I don't see this as a bug since it's a niche thing and removes counterplay.

**Comment by dragomagol** [comment]
I disagree, it doesn't make sense for a bomb to just disappear because the paper was burnt up

**Comment by SpaceVampire** [comment]
I had a similar issue before where I attached a C4 to a stool, picked up the stool, and it made the bomb vanish.

**Comment by twilightwanderer** [comment]
> I had a similar issue before where I attached a C4 to a stool, picked up the stool, and it made the bomb vanish.

The bomb, by the mechanics of implementation, disappears on any object. I haven't checked it myself yet, but another developer told me that he observed this behavior simply by changing the item icon.

**Comment by msgerbs** [comment]
Could you add a .Destroy() override for the bomb that moves it to the tile the object is on?
